piksi_tools/console/settings_view.py

```diff
@@ -14,15 +14,12 @@
 import threading
 import time
 import configparser
-from enum import IntEnum, unique
+
+from queue import Queue
```

Comment:
This is one of the modules renamed in python 2.7->3. Did you test both versions? For safety, `from six.moves.queue import Queue`

Reply:
Yes it passes tests for both in CI. But I'll make the change.

Reply:
IIRC `Queue` is Python2 only. `queue` works for both.